Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:latest
```

Comment:
Shall we specify the alpine version?

Reply:
Let's avoid the latest tag and instead use the latest alpine version?

Reply:
Agree... Added changes to PR